### PR TITLE
feat(payment-paypal): Add paypal payment method

### PR DESCRIPTION
**Pull request by JumpLink**
* Added PayPal icon
* Added PayPal translations
* Moved `PaymentSource` interfaces to common
* Updated `@beabee/beabee-common` to `0.20.3-alpha.2`
* Show PayPal Email address on account and admin contact overview pages

Related:
* https://github.com/beabee-communityrm/beabee/pull/384
* https://github.com/beabee-communityrm/beabee-common/pull/14

After merge:
* Merge the translations in the spread sheets branch tab

Preview:
Bildschirmfoto vom 2024-03-26 17-39-01

image

image

image

### CODE REVIEW

**Comment by wpf500** [comment]
Looks good so far. I think you will also need to update the `PaymentSource` component, which shows the user their current payment source on their contribution page. It should reflect the PayPal payment source you added to the backend

**Review by wpf500 — CHANGES_REQUESTED** [review]
Looks good so far. You will also need to update the `PaymentSource` component, which shows the user their current payment source on their contribution page. It should reflect the PayPal payment source you added to the backend

**Comment by JumpLink** [comment]
@wpf500 I have now moved `PaymentSource` to common because it's used from the frontend and the backend. For this, both projects must now use the latest common version, so I have also updated the frontend to `0.20.3-alpha.2`. There were a few places where typescript complained about the types, so I didn't invest too much time in that and made it easy for myself with the types for now.

I also added the paypal email beside the paypal icon on account and contact overview pages.

File: src/type/contribution-info.ts [hunk 1]
```diff
@@ -2,9 +2,10 @@ import type {
   ContributionPeriod,
   ContributionType,
   MembershipStatus,
+  PaymentSource,
 } from '@beabee/beabee-common';
 
-import type { PaymentSource } from '@type';
+import type {} from '@type';
```
Comment (wpf500) [inline src/type/contribution-info.ts]: Delete this line

**Review by wpf500 — COMMENTED** [review]
Good stuff. Just need to update the beabee-common version number once you've released v0.20.3 then this is good to go

**Review by wpf500 — APPROVED** [review]

**Pull request merged**